resources/views/members/form.blade.php

```diff
@@ -129,6 +129,13 @@
 	</div>
 </div>
 
+<div class="form-group">
+	{!! Form::label('skills', 'Vaardigheden & interesses') !!}
+	{!! Form::select('skills[]', \App\Skill::formArray(), null, ['class' => 'form-control', 'multiple' => 'multiple',
```

Comment:
Wow this works? That's amazing

Reply:
The trick is the `formSkillsAttribute` method on `App\Member`, see https://laravelcollective.com/docs/6.0/html#form-model-accessors. I discovered it during the writing of this PR, wish I'd known sooner!